woocommerce/pinterest-for-woocommerce
# Show proper error if merchant is connected to a different platform

### Changes proposed in this Pull Request:

Closes #364.

This PR adds a proper error when the merchant is connected to a different e-commerce platform (like Shopify).


### Screenshots:

![image](https://user-images.githubusercontent.com/40774170/158844926-7cfdde94-435b-474e-9ccd-2e396257e81e.png)


### Detailed test instructions:
1. Create a new Pinterest business account or have an account with no stores connected.
2. Go to Shopify, create a store, install Pinterest app and connect your business account.
3. Create a store in WooCommerce, install the plugin and connect your business account.
4. A clear error should be prompted as shows the attached picture.

### Additional details:

### Changelog entry

> Add - Clear error when the merchant is connected to another e-commerce platform.
